frontend/src/metabase/visualizations/lib/exports-branding-utils.tsx

```diff
@@ -0,0 +1,126 @@
+import ReactDOMServer from "react-dom/server";
+import { t } from "ttag";
+
+import BrandingLogo from "metabase/public/components/EmbedFrame/LogoBadge/metabase_logo_with_text.svg?component";
+
+type FooterSize = "xs" | "s" | "m" | "l" | "xl" | "xxl" | "xxxl";
+
+export const getFooterSize = (width: number): FooterSize => {
+  const sizes: [number, FooterSize][] = [
+    [200, "xs"],
+    [600, "s"],
+    [960, "m"],
+    [1280, "l"],
+    [1600, "xl"],
+    [1920, "xxl"],
```

Comment:
not a blocker, but IMHO, anything above `l` is obnoxious, even on a huge screen

. | .
--|--
l | ![l](https://github.com/user-attachments/assets/d0793e23-f8b2-40ed-b943-9df82adc62a9)
xl | ![xl](https://github.com/user-attachments/assets/2db9d4ae-9e5d-4929-99a1-f53fd8c10ca0)
xxl | ![xxl](https://github.com/user-attachments/assets/1822b50c-c78f-43e7-9903-2b86faad32f9)
xxxl | ![xxxl](https://github.com/user-attachments/assets/1556a245-a4ab-4f24-9978-58ca3c043a19)